packages/cloud2edge/Chart.yaml

```diff
@@ -11,7 +11,7 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.0.5
+version: 0.0.6
```

Comment:
since there is no single app, we have so far kept the version and appVersion in sync. Otherwise I wonder, which application (Hono, Ditto, hawkBit) the *appVersion* refers to ...

Reply:
hm, good point. changed.